### PR TITLE
new equity inclusion accessibility page

### PR DESCRIPTION
**Pull request by maneesha**
creates new EIA page

### CODE REVIEW

**Comment by maneesha** [comment]
@brynnelliott  this is missing the link to the accommodation request form.  Currently there is a `#` as a placeholder.  Once you have that ready, you can update the link and this should be ready to merge.

**Comment by brynnelliott** [comment]
@maneesha THANK YOU!! 
We have a little more testing on the form and then I will add it.

**Comment by maneesha** [comment]
OK.  I am marking this PR as draft for now.  @brynnelliott please leave a note here when it is ready.
(And all the setup you did in the issue you opened was a huge help!)

File: eia.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+## Accessibility Statement 
+
+The Carpentries is committed to providing inclusive and accessible content that enables all individuals, including those with disabilities, to participate and engage fully. We are actively working to increase the accessibility and usability of Carpentries’ content and in doing so adhere to many of the available standards and guidelines including those from the World Wide Web Consortium. 
+
+If there are any aspects of our community including the website, community discussions, instruction, or the design of our lessons that result in barriers to your inclusion, please [email us](mailto:{{site.contact}}) with as much information as possible including the name, location, and date of the event you attended, or any relevant links that provide context (e.g. Slack, GitHub). 
+
+
+## Request an Accommodation 
+
+To request an accommodation for a Carpentries event, please fill out the [accommodation request form](#).
```
Comment (brynnelliott) [inline eia.md]:
```suggestion
To request an accommodation for a Carpentries event, please fill out the [accommodation request form](https://carpentries.typeform.com/to/B2OSYaD0). 
```

**Review by brynnelliott — COMMENTED** [review]
added link to accommodation request form

**Comment by brynnelliott** [comment]
@maneesha It's ready!

**Pull request merged**